crates/runtime/src/datafusion/query.rs

```diff
@@ -277,6 +282,28 @@ impl Query {
     }
 }
 
+fn parameter_schema_for_plan(plan: &LogicalPlan) -> Result<Option<Schema>, DataFusionError> {
+    let parameters = plan
+        .get_parameter_types()?
+        .into_iter()
+        .map(|(name, dt)| {
```

Comment:
Could you please add a note when this can happen and why it is safe to proceed with this assumption

Reply:
I don't feel confident this is a great assumption in all cases. From the examples I've tested so far, this only occurs for LIMIT placeholders.